### ISSUE TITLE
Semi-Anonymous Posting

### ISSUE THREAD

**Issue by willdoran** [ushahidi/platform#1571]
- [x] Visitors to a deployment should be able to create posts without having to become full deployment users.
- [ ] It should be possible to collect some details about the user such as:
   - [ ] Name
   - [ ] Email
   - [ ] Organisation

This feature existed previously but was wiped out by one of the post rewrites.

Aha! Link: https://ushahiditeam.aha.io/features/PROD-639

**Comment by rjmackay** [comment]
This might be a dupe of #350

**Comment by rjmackay** [comment]
Related to #1399

**Comment by willdoran** [comment]
closing in favour #1399

**Comment by rjmackay** [comment]
Reopening because this is already partially implemented and #1399 is far too broad in scope IMO.

@jshorland can you review this and #1399. A minimal version of this is implemented in https://github.com/ushahidi/platform-client/pull/682 for the Kuwait deployment. But we need to decide if it goes into core. If it does, should we give users an option to enabled/disable/require author info?

**Comment by Erioldoesdesign** [comment]
Adding in comments from my MOZOL 6 project (finally)

Platform user says: They currently ask demographic questions as part of their survey set up but indicate it would be useful to be able to collect this in a protected way. BWV never want a responder to become a 'user' e.g. sign up for an account on the deployment as the nature of the responses/posts are typically one off or even if there was multiple responses/posts by one person then they do not need to be connected yet.
Responders don't tend to check back after posting once or multiple times unless they experience another issue. Follow up is hard so a way of following up with a responder reliably and privately would be useful

**Comment by rowasc** [comment]
> Visitors to a deployment should be able to create posts without having to become full deployment users.
This is already possible. 
About the PII part: I think this is better solved through fields that only non-logged in users would see (but now I'm basically going into conditional branching... which is not something we do) . Eriol's feedback is super useful here, the deployer was actually doing something like that without conditional branching, but finding it hard to follow up... so maybe something that lets you say "This is contact information" and contacting people through it... IDK. Need to think more about this.

**Comment by Angamanga** [comment]
Archive this!
